docs/sym.sh

```diff
@@ -117,12 +117,8 @@ ensureBrew() {
 
 ensurePipx() {
   if ! hasCommand pipx; then
-    if hasCommand brew; then
-      brew install pipx
-    else
-      $(getPythonPath) -m pip install --user pipx
-      $(getPythonPath) -m pipx ensurepath
-    fi
+    $(getPythonPath) -m pip install --user pipx
```

Comment:
if they have brew we should use it, as it is the recommended way to install

Reply:
If I read you correctly, then the original code was already doing the right thing, right? Or did I miss something?

Reply:
oops yup you're totally right. I read the future and did this 11 days ago: https://github.com/symopsio/homebrew-tap/commit/ac3a8be9c9eaa34e1596bfd03ed2ca45aec912cb#diff-81232f0b8715ed74b74a0539a216de91

Reply:
totally my bad for not paying attention when I created the ticket!